engine/runtime-language-arrow/src/test/resources/application-test.conf

```diff
@@ -0,0 +1 @@
+akka.coordinated-shutdown.run-by-actor-system-terminate = off
```

Comment:
Probably OK to just remove this line, and have an empty `application-test.conf`. No `akka` used in this project.